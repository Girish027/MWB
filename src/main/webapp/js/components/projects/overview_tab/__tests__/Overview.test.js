import React from 'react';
import { mount, shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

const Overview = require('components/projects/overview_tab/Overview').default;

describe('Overview', () => {
  const props = {
    models: null,
    loadingModels: false,
    projectId: '5',
    isAnyDatasetTransformed: true,
    clientId: '101',
    history: [],

  };
  let wrapper;
  afterAll(() => {
    jest.clearAllMocks();
  });
  test('should render overview component correctly with given props', () => {
    wrapper = shallow(<Overview
      {...props}
    />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
