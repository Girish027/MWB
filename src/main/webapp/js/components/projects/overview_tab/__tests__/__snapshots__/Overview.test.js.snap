// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview should render overview component correctly with given props 1`] = `
<Route>
  <Component />
</Route>
`;
